### PR TITLE
Sekindo Prebid Adapter :

### PR DESCRIPTION
**Pull request by sekindo**
A new Sekindo Adapter:
1. The Adapter was tested and verified to work ok. (built with Gulp). 
We red the latests pusg requests reviews and applied them (such as use only https, not use  bidmanager.setExpectedBidsCount and few more)

2. Added gpt integration example that was also tested using DFP.

Thanks,
Sekindo dev team

### CODE REVIEW

File: src/adapters/sekindo.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
+          bid.callback_uid = callbackId;
+          bid.bidderCode = bidCode;
+          bidmanager.addBidResponse(placementCode, bid);
+        }
+      }
+    }
+    else
+    {
+      utils.logMessage('No prebid response for placement %%PLACEMENT%%');
```
Comment (mkendall07) [inline src/adapters/sekindo.js]: You need to replace  `%%PLACEMENT%%` with the actual placement name here

File: src/adapters/sekindo.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    var iframe = utils.createInvisibleIframe();
+    iframe.id = 'skIfr_'+callbackId;
+
+    var elToAppend = document.getElementById(bid.placementCode);
+    //insert the iframe into document
+    elToAppend.insertBefore(iframe, elToAppend.firstChild);
+
+    var iframeDoc = utils.getIframeDocument(iframe);
+    iframeDoc.write(html);
```
Comment (mkendall07) [inline src/adapters/sekindo.js]: Can this script be loaded asynchronously?

File: src/adapters/sekindo.js (hunk 1 above)
Comment (sekindo) [inline src/adapters/sekindo.js]: fixed

File: src/adapters/sekindo.js (hunk 2 above)
Comment (sekindo) [inline src/adapters/sekindo.js]: It cannot, we use the iframe for async.

**Comment by protonate** [comment]
I'm getting ad payloads back but not seeing CPMs higher than 0. Is that expected when using test params?

**Comment by mkendall07** [comment]
@sekindo 
Can you please add `sekindo` to `package.json` config?

File: src/adapters/sekindo.js (hunk 1 above)
Comment (mkendall07) [inline src/adapters/sekindo.js]: @sekindo 
This has not been updated in the PR.

File: src/adapters/sekindo.js (hunk 2 above)
Comment (mkendall07) [inline src/adapters/sekindo.js]: Should `document.open` first before writing.

File: src/adapters/sekindo.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      var callbackId = bidReqeust.bidId;
+      _requestBids(bidReqeust, callbackId, pubUrl);
+      //store a reference to the bidRequest from the callback id
+      //bidmanager.pbCallbackMap[callbackId] = bidReqeust;
+    }
+  }
+
+  pbjs.sekindoCB = function(callbackId, response)
+  {
+    if (typeof (response) != 'undefined' && typeof (response.cpm) != 'undefined')
```
Comment (mkendall07) [inline src/adapters/sekindo.js]: please use triple equality.

**Comment by sekindo** [comment]
Hi,
1. The example is returning cpm now.
2. committed: using triple equality
3. committed: adding sekindo to package.json

Thx

**Comment by mkendall07** [comment]
Thanks. Please update `%%PLACEMENT%%`  as well with the actual placement name.

**Comment by sekindo** [comment]
logging - fixed

File: src/adapters/sekindo.js (hunk 2 above)
Comment (sekindo) [inline src/adapters/sekindo.js]: yep. added

**Comment by protonate** [comment]
Thanks for the fixes, confirmed bids back and ads rendered, tests passing, reviewed.

**Pull request merged**